packages/expo-notifications/src/Notifications.types.ts

```diff
@@ -768,6 +768,10 @@ export type MaybeNotificationResponse = NotificationResponse | null | undefined;
 export {
   PermissionExpiration,
   PermissionResponse,
+  /**
+   * @deprecated use the `Subscription` type instead
+   * */
   EventSubscription,
+  EventSubscription as Subscription,
```

Comment:
To be honest I would do the opposite – deprecate `Subscription` in favor of `EventSubscription` which is used elsewhere. In the long term we should export this type from the `expo` package.